index.html

```diff
@@ -0,0 +1,362 @@
+            </ul>
+          </div>
+
+          <div class="seeproject">
+            <button type="button" class="see-project-btn">See Project</button>
+          </div>
+        </div>
+        <!-- single work display section ends here -->
+
+        <!-- single work display section begins here -->
+        <div class="my-work">
+          <div class="screenshot">
+            <!-- <img src="https://dummyimage.com/350X200" /> -->
+            <img src="image/works/work4.jpg" />
+          </div>
+
+          <div class="title">
+            <h1>Multi-Post Stories</h1>
+          </div>
+          <div class="sub-headings">
+            <ul>
+              <li class="bold">Canopy</li>
+              <li class="developed marked">Backend development</li>
+              <li class="year marked">2018</li>
+            </ul>
+          </div>
+
+          <div class="description">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+
+          <div class="tags">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+          </div>
+
+          <div class="seeproject">
+            <button type="button" class="see-project-btn">See Project</button>
+          </div>
+        </div>
+        <!-- single work display section ends here -->
+      </section>
+      <!-- section 2 ends Here -->
+
+      <!-- section 3 starts here -->
+      <section class="section3">
+        <div class="about">
+          <h1>About Myself</h1>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+
+          <div class="social">LET’S CONNECT</div>
+
+          <div class="icons">
+            <ul>
+              <li>
+                <a href="#"><i class="fa fa-facebook fa-2x"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa fa-twitter fa-2x"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa fa-instagram fa-2x"></i></a>
+              </li>
+
+              <li>
+                <a href="#"><i class="fa fa-github fa-2x"></i></a>
+              </li>
+
+              <li>
+                <a href="#"><i class="fa fa-linkedin fa-2x"></i></a>
+              </li>
+            </ul>
+          </div>
+        </div>
+
+        <div class="get-resume">
+          <button type="button" class="get-resume-btn">Get My Resume</button>
+        </div>
+
+        <div class="accordation">
+          <div class="title">
+            <h1>Languages</h1>
+            <div class="icon">
+              <i class="fa fa-angle-down" aria-hidden="true"></i>
```

Comment:
I am really impressed with your project so far. Keep it up. However, this sections feels cramped and not really conforming to the figma design. Kindly check the font-family, font-size, weight, padding and margins and make sure they are as close to the figma design as possible. 


Figma design
![Screenshot from 2022-09-23 15-24-33](https://user-images.githubusercontent.com/65233860/191996455-089cae23-3e1c-4ded-a093-04528be0ebeb.png)


Your design
![Screenshot from 2022-09-23 15-25-17](https://user-images.githubusercontent.com/65233860/191996570-ad20a4d1-d94d-4a18-a0d0-84cab04b4b94.png)